drivers/sensor/bosch/bme280/Kconfig

```diff
@@ -124,4 +124,9 @@ config BME280_FILTER_16
 	bool "16"
 endchoice
 
+config BME280_BMP280
+	bool "Use BME280 driver for BMP280"
+	depends on DT_HAS_BOSCH_BMP280_ENABLED
+	default y
```

Comment:
How about instead of having a Kconfig for this having the code setting a define automatically based on what is in devicetree? something like using `DT_COMPAT_GET_ANY_STATUS_OKAY`, and if both are present throw an `#error`